<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatchManagerDialog</class>
 <widget class="QDialog" name="PatchManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addPatchButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editPatchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePatchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePatchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="PatchManagerTableView" name="patchManagerTableView">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_PatchDescription">
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PatchManagerTableView</class>
   <extends>QTableView</extends>
   <header location="global">Dialog/PatchManagerTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
